Resources/public/js/views/ez-usermenuview.js

```diff
@@ -124,6 +124,10 @@ YUI.add('ez-usermenuview', function (Y) {
 
             items.push(view);
 
+            view.fire('addedToUserMenu', {
```

Comment:
I realize this should be done after the `view.addTarget` so that others components can react to that event. Also the event should be documented (example http://ezsystems.github.io/platformui-javascript-api/files/Resources_public_js_views_universaldiscovery_ez-universaldiscoverybrowseview.js.html#l122) and you should also explain in the method description that this event is fired as well.